lib/constants.ts

```diff
@@ -0,0 +1,13 @@
+const { NEXT_PUBLIC_URL } = process.env;
+
+export const isTestnetProd = () =>
```

Comment:
4 possible envs? :sweat_smile: 
We also do not set NEXT_PUBLIC_URL currently
Based on our Vercel setup discussion we might go with [NEXT_PUBLIC_VERCEL_ENV](https://github.com/Glo-Foundation/glo-wallet/pull/32/files#diff-98e56006aa8453a29bc2b8bc8e6f718b3f17ba66470f3a52a5dc5bd643482e70R62

Reply:
was based off of these discussions with @gglucass https://glodollar.slack.com/archives/C03CD2VKP2M/p1685102744412059

Reply:
yeah imo let's keep the 4 possible envs, fien to do for now.

@Syncretik the NEXT_PUBLIC_URL still needs to be changed?

Reply:
still just use already existing utils to determine if we're deployed on prod
```
export const isProd = () => process.env.NEXT_PUBLIC_VERCEL_ENV === "production";
```

Also there was added `isLocal` method which is no used at all. As I see we just want to disable pino for local env so just use this method and remove all others?

Reply:
in fairness it was hidden in a generic `utils` script :P 
I can remove